{{cookiecutter.project_slug}}/README.rst

```diff
@@ -30,15 +30,19 @@ you're developing.  Log output will be displayed in the terminal, as usual.
 Initial Setup (APPUiO + GitLab)
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+{% if cookiecutter.environment_strategy == 'dedicated' -%}
 #. Create a *production*, *integration* and *development* project at the
+{% else -%}
+#. Create a project at the
+{% endif -%}
```

Comment:
Jinja2 tags are not easy to get right. :worried:

Here is how the whole block must look like with the minuses to get indenting and blank lines right:
```jinja2
{% if cookiecutter.environment_strategy == 'dedicated' -%}
#. Create a *production*, *integration* and *development* project at the
{%- else -%}
#. Create a project at the
{%- endif %}
```